cro-clib/src/jsonrpc.rs

```diff
@@ -108,11 +110,41 @@ fn make_syncer_config(
     ))
 }
 
+use std::sync::Arc;
+use std::sync::Mutex;
+
+#[derive(Clone)]
+struct CBindingData {
+    progress_callback: ProgressCallback,
+    user_data: u64,
+}
+
+impl CBindingCallback for CBindingData {
+    fn set_user(&mut self, user: u64) {
+        self.user_data = user;
+    }
+    fn get_user(&self) -> u64 {
+        self.user_data
+    }
```

Comment:
is this used?

Reply:
in example,
it's used to tell which context it's on
```
  CroJsonRpcPtr rpc= NULL;
    cro_create_jsonrpc(&rpc, ".storage", "ws://localhost:26657/websocket", 0xab, &progress);
    cro_run_jsonrpc(rpc, wallet_req, buf, sizeof(buf), user);
    printf("response: %s\n", buf);    

    cro_run_jsonrpc(rpc, req, buf, sizeof(buf),user);
    printf("response: %s\n", buf);
```

Reply:
in example,
it's used to tell which context it's on
```
  CroJsonRpcPtr rpc= NULL;
    cro_create_jsonrpc(&rpc, ".storage", "ws://localhost:26657/websocket", 0xab, &progress);
    cro_run_jsonrpc(rpc, wallet_req, buf, sizeof(buf), user);
    printf("response: %s\n", buf);    

    cro_run_jsonrpc(rpc, req, buf, sizeof(buf),user);
    printf("response: %s\n", buf);
```